_sidebar.md

```diff
@@ -1,11 +1,23 @@
 * [What is Hydra?](/)
+* [API](https://hydra.ojack.xyz/api/)
+---
 * [Getting Started](/getting_started)
+* [Quick Reference](/quick_reference)
+---
 * Guides
```

Comment:
Add sections from hydra book:

"Performing with hydra"
"Modulation"
"Color"

Reply:
yes! about this
i wanted to talk about it next time we met actually, but spilling some thoughts here:

guides I've written are definitely technical guides. i was thinking about how to add conceptual topics (as the ones from the hydra book), should we add them to the guides section? should there be a technical guides and a conceptual guides section? maybe there are better terms for both those sections? 

excited to finish this first pass of the "technical guides" and get to the "conceptual guides" !! there's much to explore, hence the comment i made last time we met about how hydra is so unique and how exploring these sometimes obfuscated topics would be great to expand the docs and overall the bibliography around hydra